scripts/openmc-plot-mesh-tally

```diff
@@ -0,0 +1,325 @@
+        labelTally = tk.Label(self.selectFrame, text='Tally:')
+        labelTally.grid(row=0, column=0, sticky=tk.W)
+        self.tallyBox = ttk.Combobox(self.selectFrame, state='readonly')
+        self.tallyBox['values'] = [self.datafile.tallies[i].id
+                                   for i in self.meshTallies]
+        self.tallyBox.current(0)
+        self.tallyBox.grid(row=0, column=1, sticky=tk.W+tk.E)
+        self.tallyBox.bind('<<ComboboxSelected>>', self.update)
+
+        # Planar basis selection
+        labelBasis = tk.Label(self.selectFrame, text='Basis:')
+        labelBasis.grid(row=1, column=0, sticky=tk.W)
+        self.basisBox = ttk.Combobox(self.selectFrame, state='readonly')
+        self.basisBox['values'] = ('xy', 'yz', 'xz')
+        self.basisBox.current(0)
+        self.basisBox.grid(row=1, column=1, sticky=tk.W+tk.E)
+        self.basisBox.bind('<<ComboboxSelected>>', self.update)
+
+        # Axial level
+        labelAxial = tk.Label(self.selectFrame, text='Axial level:')
+        labelAxial.grid(row=2, column=0, sticky=tk.W)
+        self.axialBox = ttk.Combobox(self.selectFrame, state='readonly')
+        self.axialBox.grid(row=2, column=1, sticky=tk.W+tk.E)
+        self.axialBox.bind('<<ComboboxSelected>>', self.redraw)
+
+        # Option for mean/uncertainty
+        labelMean = tk.Label(self.selectFrame, text='Mean/Uncertainty:')
+        labelMean.grid(row=3, column=0, sticky=tk.W)
+        self.meanBox = ttk.Combobox(self.selectFrame, state='readonly')
+        self.meanBox['values'] = ('Mean', 'Absolute uncertainty',
+                                  'Relative uncertainty')
+        self.meanBox.current(0)
+        self.meanBox.grid(row=3, column=1, sticky=tk.W+tk.E)
+        self.meanBox.bind('<<ComboboxSelected>>', self.update)
+
+        # Scores
+        labelScore = tk.Label(self.selectFrame, text='Score:')
+        labelScore.grid(row=4, column=0, sticky=tk.W)
+        self.scoreBox = ttk.Combobox(self.selectFrame, state='readonly')
+        self.scoreBox.grid(row=4, column=1, sticky=tk.W+tk.E)
+        self.scoreBox.bind('<<ComboboxSelected>>', self.redraw)
+
+        # Filter label
+        boldfont = font.Font(weight='bold')
+        labelFilters = tk.Label(self.selectFrame, text='Filters:',
+                                font=boldfont)
+        labelFilters.grid(row=5, column=0, sticky=tk.W)
+
+    def update(self, event=None):
+        if not event:
+            widget = None
+        else:
+            widget = event.widget
+
+        tally_id = self.meshTallies[self.tallyBox.current()]
+        selectedTally = self.datafile.tallies[tally_id]
+
+        # Get mesh for selected tally
+        self.mesh = selectedTally.find_filter(MeshFilter).mesh
+
+        # Get mesh dimensions
+        if len(self.mesh.dimension) == 2:
+            self.nx, self.ny = self.mesh.dimension
+            self.nz = 1
+        else:
+            self.nx, self.ny, self.nz = self.mesh.dimension
```

Comment:
I'm assuming these are always expected to unpack in 3? If not, you may want to check dimension len here too.